hw1/ex2.py

```diff
@@ -0,0 +1,6 @@
+print("Введи 3-значное число")
+n = int(input())
```

Comment:
Функцию `print` использовать необязательно. Эту строку можно поместить внутри вызова функции `input`.
```suggestion
n = int(input("Введи 3-значное число"))
```